recipe/meta.yaml

```diff
@@ -13,25 +13,34 @@ source:
   {{ hash_type }}: {{ hash_val }}
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 1
+  script: pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - python
     - setuptools
-
+    - pip
+    - wheel
```

Comment:
just curious, why do we specify wheel whenever we specify setuptools, even though we're neither unpacking nor building a wheel during the build process? (or are we?)

Reply:
We do build a wheel, which is used to create the conda package.

Reply:
ok, thanks